ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_avhrr.py

```diff
@@ -0,0 +1,471 @@
+                .write_data(satzenang2)
+
+            # Sensor Centrall Frequency
+            obsspace.create_var('MetaData/sensorCentralFrequency', dtype=chanfreq2.dtype, fillval=chanfreq2.fill_value) \
+                .write_attr('units', 'Hz') \
+                .write_attr('long_name', 'Satellite Channel Center Frequency') \
+                .write_data(chanfreq2)
+
+            # Data Provider
+            obsspace.create_var('MetaData/dataProviderOrigin', dtype=ogce2.dtype, fillval=ogce2.fill_value) \
+                .write_attr('long_name', 'Identification of Originating/Generating Center') \
+                .write_data(ogce2)
+
+            # Quality: Percent Confidence - Quality Information Without Forecast
+            obsspace.create_var('MetaData/qualityInformationWithoutForecast', dtype=qifn2.dtype, fillval=qifn2.fill_value) \
+                .write_attr('long_name', 'Quality Information Without Forecast') \
+                .write_data(qifn2)
+
+            # Wind Computation Method
+            obsspace.create_var('MetaData/windComputationMethod', dtype=swcm2.dtype, fillval=swcm2.fill_value) \
+                .write_attr('long_name', 'Satellite-derived Wind Computation Method') \
+                .write_data(swcm2)
+
+            # ObsType based on computation method/spectral band
+            obsspace.create_var('ObsType/windEastward', dtype=obstype2.dtype, fillval=swcm2.fill_value) \
+                .write_attr('long_name', 'Observation Type based on Satellite-derived Wind Computation Method and Spectral Band') \
+                .write_data(obstype2)
+
+            # ObsType based on computation method/spectral band
+            obsspace.create_var('ObsType/windNorthward', dtype=obstype2.dtype, fillval=swcm2.fill_value) \
+                .write_attr('long_name', 'Observation Type based on Satellite-derived Wind Computation Method and Spectral Band') \
+                .write_data(obstype2)
+
+            # Pressure
+            obsspace.create_var('MetaData/pressure', dtype=pressure2.dtype, fillval=pressure2.fill_value) \
+                .write_attr('units', 'pa') \
+                .write_attr('long_name', 'Pressure') \
+                .write_data(pressure2)
+
+            # Height (mimic prepbufr)
+            obsspace.create_var('MetaData/height', dtype=height2.dtype, fillval=height2.fill_value) \
+                .write_attr('units', 'm') \
+                .write_attr('long_name', 'Height of Observation') \
+                .write_data(height2)
+
+            # Station Elevation (mimic prepbufr)
+            obsspace.create_var('MetaData/stationElevation', dtype=stnelev2.dtype, fillval=stnelev2.fill_value) \
+                .write_attr('units', 'm') \
+                .write_attr('long_name', 'Station Elevation') \
+                .write_data(stnelev2)
+
+            # U-Wind Component
+            obsspace.create_var('ObsValue/windEastward', dtype=uob2.dtype, fillval=wspd2.fill_value) \
```

Comment:
Could the fillval be uob2.fill_value here? Asking for the sake of consistency.

Reply:
It looks like this was a typo in the original GOES ABI converter that has been copy/pasted to every other AMV converter. I can change it here but I should probably go back to submit a small PR to fix the others.

Reply:
resolved via https://github.com/NOAA-EMC/GDASApp/pull/997/commits/df99b8f4821bac5a90dc1ad0494d35c7df3ea590